mobile_back_tflite/cpp/backend_tflite/BUILD

```diff
@@ -60,82 +60,14 @@ cc_library(
     alwayslink = 1,
 )
 
-cc_library(
-    name = "neuron_delegate",
-    srcs = select({
-        "@org_tensorflow//tensorflow:android": [
-            "@neuron_delegate//neuron/java:libtensorflowlite_neuron_jni.so",
-        ],
-    }),
-    hdrs = select({
-        "@org_tensorflow//tensorflow:android": [
-            "@neuron_delegate//neuron:APUWareUtilsApi.h",
-            "@neuron_delegate//neuron:neuron_delegate.h",
-        ],
-    }),
-)
-
-cc_library(
-    name = "tflite_neuron_c",
-    srcs = ["tflite_c.cc"],
-    hdrs = [
-        "neuron/tflite_settings_mtk.h",
-        "tflite_settings_android.h",
-        "tflite_settings_apple.h",
-        "tflite_settings_windows.h",
-        "thread_pool.h",
-        "utils.h",
-    ],
-    copts = tflite_copts() + [
-        "-Iexternal/neuron_delegate",
-    ] + select({
-        "//flutter/android/commonlibs:use_asan": [
-            "-fsanitize=address",
-            "-g",
-            "-O1",
-            "-fno-omit-frame-pointer",
-        ],
-        "//conditions:default": [],
-    }),
-    local_defines = ["MTK_TFLITE_NEURON_BACKEND"],
-    deps = [
-        "//flutter/cpp/c:headers",
-        "@org_tensorflow//tensorflow/lite/c:c_api",
-        "@org_tensorflow//tensorflow/lite/c:common",
-        "@org_tensorflow//tensorflow/core:tflite_portable_logging",
-    ] + select({
-        "@org_tensorflow//tensorflow:android": [
-            ":neuron_delegate",
-            "@org_tensorflow//tensorflow/lite/delegates/gpu:delegate",
-        ],
-        "//conditions:default": [],
-    }),
-    alwayslink = 1,
-)
-
 tflite_jni_binary(
     name = "libtflitebackend.so",
-    exported_symbols = "//flutter/cpp/c:exported_symbols.lds",
```

Comment:
@d-uzlov Removing this caused an `symbol not found` error when testing the backend via command line (https://github.com/mlcommons/mobile_app_open/blob/master/flutter/cpp/binary/README.md)
You can also clone this branch `360-add-coreml-backend` and run `cd mobile_back_apple && make tflite` to test.

Reply:
fortunately, this should be a macOS/iOs specific problem because macOS/iOS's linker (ld) doesn't support linker script.

Reply:
we can find how `tflite_jni_binary()` handles this at https://github.com/tensorflow/tensorflow/blob/r2.9/tensorflow/lite/build_def.bzl#L154-L190

for macOS, a exported symbol list, `-Wl,-exported_symbols_list,...`, is used
for others, linker script, `-Wl,--version-script,...`, is used

Reply:
@freedomtan 👍